app/src/db/models/submission.ts

```diff
@@ -65,7 +66,12 @@ export default {
       notes: input.notes,
       user: input.user?.userId ? { connect: { userId: input.user.userId } } : disconnectRelation,
       intakeStatus: input.intakeStatus,
-      applicationStatus: input.applicationStatus
+      applicationStatus: input.applicationStatus,
+      guidance: input.guidance ? input.guidance : false,
```

Comment:
cant these just be `guidance: input.guidance`
the db field defaults to false.. the html form can submit false